pkg/virt-api/api.go

```diff
@@ -520,6 +521,10 @@ func (app *virtAPIApp) composeSubresources() {
 				list.GroupVersion = version.Group + "/" + version.Version
 				list.APIVersion = version.Version
 				list.APIResources = []metav1.APIResource{
+					{
+						Name:       "expand-vm-spec",
```

Comment:
shouldn't this be `virtualmachines/expand-spec`?

Reply:
No, `expand-vm-spec` is a separate endpoint. See line 577 for `virtualmachines/expand-spec`.